index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formula 1 Get Together Conference</title>
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <div class="top-navbar">
+            <img src="images/facebook.svg" alt="" class="facebook-image">
+            <img src="images/twitter.svg" alt="" class="twitter-image">
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+        </div>
+    </header>
+
+    <header class="hamburger-logo">
+    
+            <div class="header-container">
+                <a href="#"><img src="images/mobile_menu1.svg" alt="menu-image" class="ham"></a>
+                <nav>
+                <a href="#"><img src="images/F1_logo.png" alt="summit-logo" class="logo"></a>
+                </nav>
+            </div>
+         
+
+        <nav class="menu" id="menu-pop">
+            <img src="images/Icon-close.svg" alt="" class="close-button">
+            <ul class="nav-content">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program" id="nav-grey">Program</a></li>
+                <li><a href="#join" id="nav-grey">Join</a></li>
+                <li><a href="#sponsor" id="nav-grey">Sponsor</a></li>
+                <li><a href="#news" id="nav-grey">News</a></li>
+            </ul>
```

Comment:
- [ ] Kindly make sure the navbar sticks to the Cindy design 😄 . The alignment should be the same as in the Cindy design 🛩️ 

Your design
![Capture](https://github.com/MarkYegon7/Capstone-Project/assets/80458199/02580402-e597-4a2c-ba80-a7fc5f460c6d)

Cindy design
![Capture1](https://github.com/MarkYegon7/Capstone-Project/assets/80458199/9651ab2c-f8d7-4ac9-8c86-593263ec2605)

- [ ] Make sure that all the Navlinks are present, and do not forget to customize them according to your subject 👍

Reply:
Fixed the alignment